### PR TITLE
DOC: Add notes about XADC read-out via OpenFPGALoader

### PR DESCRIPTION
**Pull request by colinoflynn**
This adds the XADC read-out for checking power stability if required

### CODE REVIEW

File: doc/guide/fpga-update.md [hunk 1]
```diff
@@ -95,10 +95,36 @@ If the `FPGA Config` LED is not coming on, this could indicate the bitstream was
 
 If the `FPGA Config` LED is on but the `Ibex Boot` LED is not, it may be that you have programmed (or selected) a different bitstream than one that runs the CHERIoT demo. Try reloading the bitstream, and try power cycling the device.
 
-### Device Rebooting During/After Programming
+### Device Rebooting During/After Programming (No Serial Activity)
 
 The Sonata board takes a fair amount of power (>500mA) from the USB interface, and should be connected via USB-C. Typically it is close enough to the USB 2.0 limits that it will work with the adapter most of the time, but if you are having reliability issues we recommend trying a different computer, ideally one with a USB-C port.
 
+If you use openFPGALoader built from the [xadc-vccs branch here](https://github.com/newaetech/openFPGALoader/tree/xadc-vccs) you can print the min/max VCCINT ranges. To do this, simply run:
```
Comment (marnovandermaas) [inline doc/guide/fpga-update.md:102]: Looks like this has been merged upstream, so we can update this link.

**Review by marnovandermaas — COMMENTED** [review]
The instructions work for me! Just update repo to upstream and then this is good to go.

```
empty
Jtag frequency : requested 6.00MHz   -> real 6.00MHz  
{"temp": 33.4599, 
    "maxtemp": 34.152, 
    "mintemp": 26.4697, 
"raw":  {"0": 39880, "1": 21883, "2": 39214, "3": 0, "4": 0, "5": 0, "6": 21814, "7": 0},
"vccint": 0.999023, 
   "maxvccint": 1.01367, 
   "minvccint": 0.988037, 
"vccaux": 1.7937, 
   "maxvccaux": 1.79663, 
   "minvccaux": 1.79004, 
}
```

**Comment by colinoflynn** [comment]
Fixed now - I changed the wording slightly as in my testing the brown-out isn't always obvious, may be some ringing that you see in the results. If people have problems however it would be a good troubleshooting step, so left it a little more open-ended until more data is collected.

**Review by marnovandermaas — APPROVED** [review]

**Comment by colinoflynn** [comment]
Also - I put a kinda awkward date timestamp in the URL, we can switch that after there is a release. I just wanted anyone who built OpenFPGALoader 2 days ago to know that they need to build a new one in case they thing a week old is new enough ;-)

**Pull request merged**